.editorconfig

```diff
@@ -16,7 +16,7 @@ trim_trailing_whitespace = false
 indent_size = 4
 
 [*.sh]
-indent_size = 4
```

Comment:
Hmm, _every_ .sh file in this project is using a 4-space indent. This change _should_ be failing CI...

Reply:
/shrug

Maybe because the `ce`-setup is using `ls-tree` with a delta-check (therefore only actually checking the new/updated `*.sh` files? Regardless, I'm not a fan of 4-space indent: can make scripts too wide, too quickly.

Reply:
I think `ec` isn't failing because the indent count is unset. Hmm, not ideal that this setting of 2 allows multiples of 2 to be valid. Need to check the options and get this enforced correctly.

It doesn't matter much to me whether the project uses a 2 space indent or 4. All that matters is that every file is consistent across the whole project. So change it everywhere if you want.

Also, if switching to `2`, since the `*` block this uses `2` already, this block for `*.sh` can just be removed.

Reply:
> Also, if switching to `2`, since the `*` block this uses `2` already, this block for `*.sh` can just be removed.

I don't know if it's migraine-fog or what, but I can't parse seem to figure out what this statement means?

Reply:
In the file, scroll up. You'll see the `*` block. It already uses space = 2. So, this `*.sh` block is not needed, if this is the only setting it contains.

Reply:
Fixed in 05b0383e048110e5371487eccf0daffdd54d1410